dolphin-dnn/src/main/java/edu/snu/dolphin/dnn/layerparam/initializer/PoolingLayerParameterInitializer.java

```diff
@@ -98,8 +98,15 @@ public int getIndex() {
     if (paddingWidth >= kernelWidth) {
       throw new IllegalArgumentException("Padding width should be less than kernel width.");
     }
-    computedShape[0] = (int) Math.ceil((float) (inputShape[0] - kernelHeight + 2 * paddingHeight) / strideHeight) + 1;
-    computedShape[1] = (int) Math.ceil((float) (inputShape[1] - kernelWidth + 2 * paddingWidth) / strideWidth) + 1;
+    if (inputShape.length == 2) {
+      computedShape[0] = (int) Math.ceil((float) (inputShape[0] - kernelHeight + 2 * paddingHeight) / strideHeight) + 1;
+      computedShape[1] = (int) Math.ceil((float) (inputShape[1] - kernelWidth + 2 * paddingWidth) / strideWidth) + 1;
+      computedShape[2] = 1;
```

Comment:
I wonder if we should change the shape length from `2` to `3`. How about maintaining the shape length?